pages/api/occurrences.js

```diff
@@ -43,6 +43,11 @@ export default async (req, res) => {
     }
 
     const listOccurrencesResponse = await response.json();
+
+    if (!listOccurrencesResponse.occurrences?.length) {
+      return res.status(StatusCodes.OK).send(null);
```

Comment:
Should this return `{"occurrences": []}` ? I know there's an issue at the moment with listing occurrences when there's no data, but I'd expect that to be the response once that issue is fixed.

Reply:
I decided to return null here so I could just [check for the existence of data](https://github.com/rode/rode-ui/blob/001b530ec5b43aa01bba8974e51d36b70c045fed/components/resources/ResourceOccurrences.js#L42) in order to determine whether or not a resource was found instead of checking for the length of the mapped occurrences. I am assuming we will never have a resource that doesn't have occurrences, since that is how the resources are initially pulled, so I figured this was an okay way to distinguish for now.

Reply:
But I see what you're saying.. really I could move this logic into that component that I linked above. I am a little hesitant to do that because the mapping separates the occurrences into their sections so I'll have to have a more complicated check on the client to determine if the resource exists, but I can totally update that if you think this is going to be confusing in the future.